src/chrome/extension/scripts/background.ts

```diff
@@ -41,15 +44,33 @@ chrome.runtime.onMessageExternal.addListener(
         return true;
     });
 
-// Launch the Chrome webstore page for the uProxy app.
+// Launch the Chrome webstore page for the uProxy app,
+// or activate the user's tab open to uproxy.org/chrome-install
 function openDownloadAppPage() : void {
-  chrome.tabs.create(
-      {url: 'https://chrome.google.com/webstore/detail/uproxyapp/fmdppkkepalnkeommjadgbhiohihdhii'},
-      (tab) => {
-        // Focus on the new Chrome Webstore tab.
-        chrome.windows.update(tab.windowId, {focused: true});
+  chrome.windows.get(mainWindowId, {populate: true},
+      (windowThatLaunchedUproxy) => {
```

Comment:
can you change the indentation of this to be:
```
  chrome.windows.get(mainWindowId, {populate: true}, (windowThatLaunchedUproxy) => {
    if (windowThatLaunchedUproxy) {.....
```
or even 
```
  chrome.windows.get(mainWindowId, {populate: true},
      (windowThatLaunchedUproxy) => {
    if (windowThatLaunchedUproxy) {.....
```
that way the if statement is only 2 spaces in chrome chrome.windows.get, rather than 6?